db/migrate/20210304185255_add_status_to_subscription.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToSubscription < ActiveRecord::Migration[6.0]
+  def change
+    add_column :subscriptions, :status, :integer
```

Comment:
if you do want to stick with a non-nil status, your `after_initialize` block can be changed to 
![image](https://user-images.githubusercontent.com/1001691/112728670-fe504980-8eed-11eb-9e8b-14b021952466.png)

Reply:
changed to nil 👍🏼